iphone/Classes/TiUIiOSStepper.m

```diff
@@ -0,0 +1,204 @@
+}
+
+-(void)setDecrementSelectedImage_:(id)value
+{
+    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:NO];
+    [[self stepper] setDecrementImage:[self loadImage:value] forState:UIControlStateSelected];
+}
+
+-(void)setDecrementDisabledImage_:(id)value
+{
+    [[self stepper] setDecrementImage:[self loadImage:value] forState:UIControlStateDisabled];
+}
+
+#pragma mark Increment backgroundImage
+-(void)setIncrementImage_:(id)value
+{
+    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:NO];
+    [[self stepper] setIncrementImage:backgroundImageCache forState:UIControlStateNormal];
+}
+
+-(void)setIncrementSelectedImage_:(id)value
+{
+    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:NO];
+    [[self stepper] setIncrementImage:backgroundImageCache forState:UIControlStateSelected];
+}
+
+-(void)setIncrementDisabledImage_:(id)value
+{
+    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:NO];
+    [[self stepper] setIncrementImage:backgroundImageCache forState:UIControlStateDisabled];
+}
+
+-(void)frameSizeChanged:(CGRect)frame bounds:(CGRect)bounds
+{
+    [TiUtils setView:[self stepper] positionRect:bounds];
+    [super frameSizeChanged:frame bounds:bounds];
+}
+
+-(CGFloat)contentWidthForWidth:(CGFloat)value
+{
+    return [[self stepper] sizeThatFits:CGSizeMake(value, 0)].width;
+}
+
+-(CGFloat)contentHeightForWidth:(CGFloat)value
+{
+    return [[self stepper] sizeThatFits:CGSizeMake(value, 0)].height;
+}
+
+-(BOOL)hasTouchableListener
+{
+    // since this guy only works with touch events, we always want them
+    // just always return YES no matter what listeners we have registered
+    return YES;
+}
+
+- (IBAction)stepperChanged:(id)sender
+{
+    NSNumber *newValue = [NSNumber numberWithDouble:[[self stepper] value]];
+    [self.proxy replaceValue:newValue forKey:@"value" notification:YES];
+    
+    if ([self.proxy _hasListeners:@"change"]) {
+        [self.proxy fireEvent:@"change" withObject:[NSDictionary dictionaryWithObject:newValue forKey:@"value"]];
+    }
+}
+
+- (IBAction)stepperTouch:(id)sender forEvent:(UIEvent *)event
+{
+    UITouch *touch = [[event allTouches] anyObject];
+    if([self.proxy _hasListeners:@"click"]) {
+        [self.proxy fireEvent:@"click" withObject:[NSDictionary
+                                                   dictionaryWithDictionary:[TiUtils touchPropertiesToDictionary:touch
+                                                                                andPoint:[touch locationInView:self]]]];
+    }
+}
+
+
+-(UIImage *)imageWithImage:(UIImage *)image fullScale:(bool)full {
+    //UIGraphicsBeginImageContext(newSize);
+    // In next line, pass 0.0 to use the current device's pixel scaling factor (and thus account for Retina resolution).
+    // Pass 1.0 to force exact pixel size.
+    [self stepper];
```

Comment:
Thats there uncase this somehow gets called while the stepper is nil , which did happen and caused crash, this way if nil it will be initialized.